sdk/stable-layout/src/stable_vec.rs

```diff
@@ -24,27 +28,22 @@ use std::{marker::PhantomData, mem::ManuallyDrop, ptr::NonNull};
 /// ```
 #[repr(C)]
 pub struct StableVec<T> {
-    pub ptr: NonNull<T>,
-    pub cap: usize,
-    pub len: usize,
+    pub addr: u64,
```

Comment:
`ptr` was renamed to `addr` here but forgotten to change it in `test_memory_layout()` as well.